examples/esp-idf-component/main/main.c

```diff
@@ -0,0 +1,248 @@
+#include <stdio.h>
```

Comment:
I think this (and micrordklib-idf-component) could maybe use a README? Some instructions for building and running?